lib/lightning/work_orders.ex

```diff
@@ -447,33 +447,35 @@ defmodule Lightning.WorkOrders do
           | UsageLimiting.error()
           | {:error, :enqueue_error}
   def retry_many([%WorkOrder{} | _rest] = workorders, opts) do
-    runs_ids =
-      workorders
-      |> Enum.map(& &1.id)
-      |> get_first_runs_for_retry()
-
     with project_id <- Keyword.fetch!(opts, :project_id),
          :ok <-
            UsageLimiter.limit_action(
-             %Action{type: :new_run, amount: length(runs_ids)},
+             %Action{type: :new_run, amount: length(workorders)},
```

Comment:
Before checking the allowed amount, it needs to filter for only the workorders having an available dataclip (wiped_at = nil).